File: C/EX14.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  int a, b, c;
  
  printf("Digite o primeiro valor: ");
  scanf("%i", &a);
 
  printf("Digite o segundo valor: ");
  scanf("%i", &b);
 
  printf("Digite o terceiro valor: ");
  scanf("%i", &c);
 
  if (a > b)
  {
      if (a > c)
      {
          printf ("%i e o maior", a);
      }
      else 
      {
          printf ("%i e o maior", c);
      }
  }
    else if(b > c)
    {
      printf ("%i e o maior", b);
    }
     else
    {
      printf ("%i e o maior", c);
     }
 
  return 0;
}

File: Python/Ex14.py
a = float(input("Digite o primeiro valor: "))
b = float(input("Digite o segundo valor: "))
c = float(input("Digite o terceiro valor: "))

if (a > b):

    if (a > c):
        print ("O valor", a, "é o maior")
    else:
        print ("O valor", c, "é o maior")

elif b > c:
    print ("O valor", b, "é o maior")
else:
    print ("O valor", c, "é o maior")


